azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -194,8 +194,8 @@ public AzkabanWebServer(Server server, Props props) throws Exception {
     // TODO remove hack. Move injection to constructor
     executorManager = SERVICE_PROVIDER.getInstance(ExecutorManager.class);
```

Comment:
At some point, we would like to get rid of these custom injections as well. Looks Ok for now.